.github/workflows/lint.yml

```
@@ -0,0 +1,22 @@
+name: Lint
+
+on: [pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: '.node-version'
+
+      - name: Install dependencies
+        run: npm install
```

テストであればpacakge-lock.jsonを参照する `npm ci` をつかっておくほうがよさそうです。
https://docs.npmjs.com/cli/v9/commands/npm-ci

```suggestion
        run: npm ci
```

---

なるほど！